README

```diff
@@ -320,9 +320,13 @@ environmentMaxTribes: int
     Set the maximum number of tribes in the starting population.
     Default: 0
 
-environmentPollutionDiffusionDelay: int
```

Comment:
This option should not be removed since it does something different than the added features. It sets the timestep interval for when diffusion should happen (e.g. every 10 timesteps).

Reply:
You're right, that was an accident. I meant to remove the ones that I added.